test/integration/sagemaker/test_mnist.py

```diff
@@ -20,6 +20,8 @@
 
 from sagemaker_tensorflow_container.training import SAGEMAKER_PARAMETER_SERVER_ENABLED
 
+MNIST_DATA_S3_PATH = 's3://sagemaker-sample-data-us-west-2/tensorflow/mnist'
```

Comment:
This location won't work in other regions. Let's upload the data like the line 77 does.

Reply:
Currently I think this test is only run in us-west-2. But it's a good point. This data is actually uploaded in all the regions. I will put the region in the path.